### DISCUSSION TITLE
Welcome to m3u-stream-merger-proxy Discussions!

### DISCUSSION THREAD

**Discussion by sonroyaalmerol** [sonroyaalmerol/m3u-stream-merger-proxy#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jaimeaerazo11** [comment]
Idea! 
If we can exclude files by group and by title, why not by format? Example: exclude MP4 and MKV files

**Reply by sonroyaalmerol** [reply]
Hi! Unless there's a standard/reliable way of doing so from just reading and parsing an M3U, that would be outside the scope of this project.
